src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BedSearchService.kt

```diff
@@ -92,27 +100,58 @@ class BedSearchService(
     )
   }
 
+  @Suppress("detekt:CyclomaticComplexMethod")
   fun findTemporaryAccommodationBeds(
     user: UserEntity,
-    probationDeliveryUnit: String,
+    probationDeliveryUnit: String?,
```

Comment:
This parameter will be removed after the UI work is done

Reply:
is it worth marking at least one of these occurrences of probationDeliusUnit as @deprecated in code in case the removal doesn't happen for a while. It would help anyone reading the code understand why there are multiple ways to use provide the PDU Info?